mariadb/manifest.jsonnet

```diff
@@ -51,25 +51,96 @@ kpm.package({
   resources: [
     // Config maps.
     {
-      file: "configmaps/my.cnf.yaml.j2",
-      template: (importstr "templates/configmaps/my.cnf.yaml.j2"),
-      name: "mariadb-mycnf",
+      file: "configmaps/seed.sh.yaml.j2",
+      template: (importstr "templates/configmaps/seed.sh.yaml.j2"),
+      name: "seedsh",
```

Comment:
Could we keep the `mariadb` prefix? We already have over 50 configmaps: it would get messy/confusing without it and conflicts might happen (especially for the `startsh` below!).